pkgs/development/libraries/capnproto/default.nix

```
@@ -9,6 +9,11 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-soBUp6K/6kK/w5LI0AljDZTXLozoaiOtbxi15yV0Bk8=";
   };
 
+  nativeBuildInputs = [ cmake ]
+    ++ lib.optional (!(stdenv.hostPlatform.isCompatible stdenv.buildPlatform)) capnproto;
```

```suggestion
    ++ lib.optional (!stdenv.hostPlatform.isCompatible stdenv.buildPlatform) capnproto;
```

Are those equivalent?